### ISSUE TITLE
Design intent Decision Tools - Find a VA Form PDF Download Instructions

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#31832]
# Steps to complete Design Intent Collaboration

## Before meeting

**VFS Product Manager or Designer**
- [x]  Create this issue and notify the Collaboration Cycle team of this ticket in the in #vfs-platform-support ([see Slack Workflow (User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)). 
- [x] Add artifacts listed below to the ticket at least 2 days before the scheduled meeting.

**Platform**
- [x] Schedule meeting with Platform reviewers and VFS team attendees listed in ticket. 

## After meeting

**Platform**
- [x] Create feedback tickets and link to the Design Intent epic (if no feedback, please comment stating that). 
- [x] Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061).

**VFS**
- [ ] Respond to Platform feedback tickets.
- [ ] Close this ticket when review is complete and feedback has been addressed. 

### Has your product been reviewed by the OCTO-DE Design Lead?
- [X] Yes
- [ ] No

## Artifacts

Provide links or documents for the following: 

- [x] [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/README.md)
 
- [x] Initial Mockups:  https://user-images.githubusercontent.com/68560244/134392860-7bd260f9-4b15-4621-ad26-add3ddda8a0c.png
- [X] Mockups with finalized copy: https://www.sketch.com/s/6e7784a1-5b01-4aec-9ec7-4bbf788d08fa/a/Mypa8g7
- [x] Use cases
    - [Find a Form Search Landing Page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/mvp/design/find-va-form-use-cases.md)
        - Use cases [S-F](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/mvp/design/find-va-form-use-cases.md#use-case-s-f) and [S-G](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/mvp/design/find-va-form-use-cases.md#use-case-s-g) are pertinent to this design intent.
    - [Find a Form Detail Page](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/mvp/design/find-va-form-use-cases.md#use-case-d-f)
        - Use cases [D-F](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/mvp/design/find-va-form-use-cases.md#use-case-d-f) and [D-G](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/mvp/design/find-va-form-use-cases.md#use-case-d-g) are pertinent to this design intent.

Note - this modal is the product of [user research recommendations](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/find-a-va-form/post-mvp-releases/research/research-findings.md#recommendations).    After users failed to notice inline info-text and notes related to Adobe Reader, Marci met with Ryan Thurwell and Angela Fowler, and they agreed that this was a good use case for a modal to draw needed attention.    In that meeting, a high-level design was agreed upon, and a [mockup was created and has been iterated on several times](https://github.com/department-of-veterans-affairs/va.gov-team/issues/29704).  Marci took this design to Collab office hours on 9/29 for feedback.   

Final updates to the  mockup were on put hold pending final copy recommendations from the Content team, which were [received on Oct 17](https://github.com/department-of-veterans-affairs/va.gov-team/issues/30675#issuecomment-945160596), but cannot be updated in mockup until current sprint due to design prior commitments.

### Required
- Product Manager: Marci McGuire, Linda Li will be taking over
- OCTO-DE Product lead: Dave Conlon
- OCTO-DE Design lead: Ryan Thurlwell

### Team Members

- Cassandra Allen

### VFS Attendees

- Content and IA:  Mikki
- Accessibility:  Angela Fowler, Josh Kim
- QA specialists: Tze

## Scheduling

**Tue, 10/26** **11:00 AM ET**

## Additional information
For the Design Intent, you'll receive feedback from design and IA only. You won't receive feedback from accessibility, content, or QA, as it's too early in the development process.

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Design Intent Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Design-intent.1782153235.html) on Platform website for more information about the Collaboration Cycle.

**Comment by MarciMcGuireGCIO** [comment]
Thank you @allison0034! :)

**Comment by LindaLi123** [comment]
Hello @MarciMcGuireGCIO Since Beth confirmed the instructions, do we need to do anything else before we can close this ticket?

**Comment by MarciMcGuireGCIO** [comment]
@LindaLi123 this morning I was still seeing comments in another ticket related to this, so not sure this is completely finished.  I will close it at EOD tomorrow, and we will go with whatever the content is at that time.

**Comment by bethpottsVADEPO** [comment]
@MarciMcGuireGCIO Which other ticket?

**Comment by MarciMcGuireGCIO** [comment]
@bethpottsVADEPO -- https://github.com/department-of-veterans-affairs/va.gov-team/issues/30421#issuecomment-956239284 but now re-reading I realize Laura was working on the R&S article.

**Comment by MarciMcGuireGCIO** [comment]
This feedback has been incorporated into our mockups
